src/main/java/com/engine/GameEngine.java

```diff
@@ -216,6 +216,8 @@ private String getLookResult(String objectToFind) {
 
     public StringBuilder showStatus(String location) {
         StringBuilder builder = new StringBuilder();
+        System.out.println(player.getAreasVisited());
```

Comment:
Was this meant to be appended to builder? or were these for testing?

Reply:
The getAreasVisited is for visiting

Reply:
Okay wasn't sure if these two print statements were needed